packages/e2e-tests/specs/performance/post-editor.test.js

```diff
@@ -151,6 +154,58 @@ describe( 'Post Editor Performance', () => {
 		}
 	} );
 
+	it( 'Typing within containers', async () => {
+		// Measuring block selection performance.
+		await createNewPost();
+		await loadHtmlIntoTheBlockEditor(
```

Comment:
Would it make sense to instead add this test in another `describe`, so we don't have to go through the setup steps twice? I can't think offhand of any other container-specific tests we might want to run but it's possible there may be others in the future.

Reply:
I thought about this a bit and I'm not entirely sure what's the best solution here. Ideally each `it` is independent. Right now, some of them are:

 - typing in container, selecting blocks as both create new posts in the beginning
 - the remaining ones are not independent in the sense that they use whatever is loaded before the test starts (and it seems to be different across metrics depending on where they are placed).

So I do think there's something to be done to improve how things are organized and make it consistent but let's leave this for its dedicated PR and address it for all the tests and not just this particular one.